src/constants/xdai/index.ts

```diff
@@ -0,0 +1,73 @@
+import { Token } from '@uniswap/sdk-core'
+import { SupportedChainId as ChainId } from '../chains'
+
+// native constants
+export const XDAI_SYMBOL = 'XDAI'
+export const XDAI_NAME = 'xDai'
+
+// xDAI tokens
+export const WXDAI = new Token(
```

Comment:
We'll only need wxDai token address, everything else is not used.

Or do you intend to use it in follow up PRs?